### PR TITLE
Welcome Email

### PR DESCRIPTION
**Pull request by MrBeardface**
- Change content for email
- Adjust spacing
- Remove H1

### CODE REVIEW

**Comment by MrBeardface** [comment]
Current Issues that need to be rendered:

1. Button needs to be fixed.
2. Image size needs to be adjusted.
       * Change margin-top to 10px
       * Change margin-bottom to 30px

File: app/views/user_mailer/welcome_email.html.erb [hunk 1]
```diff
@@ -1,13 +1,27 @@
...
+    </td>
+  </tr>
+  <tr>
+    <td class="content-block">
+      <p>You have tips. We have tips. Let's put them on Rails.</p>
+    </td>
+  </tr>
+  <tr>
+    <td class="content-block">
+       <a href="#" class="btn-primary"><%= link_to "Where's this lead?", new_tip_path %></a>
```
Comment (ohlincik) [inline app/views/user_mailer/welcome_email.html.erb]: Where is this button supposed to go? Do we really need it?

**Comment by ohlincik** [comment]
I made the adjustments discussed above. I'm not sure where the button in the email is supposed to go and whether we really need it there.

**Comment by MrBeardface** [comment]
The button is unnecessary. I vote to remove it.

**Pull request merged**